src/ui/registration/components/entropy.tsx

```diff
@@ -31,35 +27,38 @@ const styles = StyleSheet.create({
   bigFont: {
     fontSize: Typography.text4XL,
   },
+  contentView: {
```

Comment:
A lot of these keys are duplicated in the style definition for the `Container` component.

Reply:
Addressed in a comment.